### PR TITLE
Character display and limit

### PR DESCRIPTION
**Pull request by Joncitone**

### CODE REVIEW

File: client/components/NewEntryForm.js [hunk 1]
```diff
@@ -67,6 +83,7 @@ class NewEntryForm extends Component {
               type="text"
               onChange={this.handleChange}
               value={this.state.content}
+              inputProps={{ maxLength: 5000 }}
```
Comment (jsmney) [inline client/components/NewEntryForm.js]: this is the same value as characterLimit, can you substitute the variable here?

**Review by jsmney — DISMISSED** [review]
approving but request you make the one change first.

File: client/components/NewEntryForm.js (hunk 1 above)
Comment (Joncitone) [reply client/components/NewEntryForm.js]: updated to this.state.characterLimit

**Review by Joncitone — COMMENTED** [review]
LGTM!

**Review by jsmney — APPROVED** [review]

**Pull request merged**